test/functional/cmdLineTests/SystemPropertiesTest/playlist.xml

```diff
@@ -67,4 +67,26 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 			<impl>ibm</impl>
 		</impls>
 	</test>
+	<test>
+		<testCaseName>cmdLineTester_SysPropRepeatDefinesTest</testCaseName>
+		<variations>
+			<variation>NoOptions</variation>
+		</variations>
+		<command>$(JAVA_COMMAND) $(JVM_OPTIONS) \
+			-DJARPATH=$(Q)$(TEST_RESROOT)$(D)SystemPropertiesTest.jar$(Q) \
+			-DEXE=$(SQ)$(JAVA_COMMAND) $(JVM_OPTIONS) $(SQ) \
+			-jar $(CMDLINETESTER_JAR) -config $(Q)$(TEST_RESROOT)$(D)sysproprepeatdefinestest.xml$(Q) \
+			-nonZeroExitWhenError; \
+			$(TEST_STATUS)</command>
+		<levels>
+			<level>sanity</level>
```

Comment:
This is unlikely to get broken by other changes, it can run in extended.

Reply:
Changed to `extended`.